### DISCUSSION TITLE
Better notification when "mute myself" is on.

### DISCUSSION THREAD

**Discussion by MrrGlen** [jamulussoftware/jamulus#2056] [UI Related]
Lost count of the times I have "mute myself" turned on an played a whole song/songs and only realised after that I was the only one that could hear it! 
Could there be a better notification that you have "mute myself" on, maybe your channel could turn red or something. Need a better reminder than just the small red light in the bottom left.

**Comment by rdica** [comment]
What version of the client are you using?
in 3.8.0 I see a bright red strip along the top of the clients main window when I Mute Myself. Can't miss it:

![Screenshot_2021-10-01_14-57-23](https://user-images.githubusercontent.com/16609801/135672861-8b242890-2fbd-4d29-82b0-fe3c67dc7701.png)

**Reply by MrrGlen** [reply]
Ha yeah forgot about that, think it's where it's on my second screen and full screen so right up the top where I'm not looking. Have made the window alot smaller and changed the theme notice it alot more now! Thanks.

**Reply by gilgongo** [reply]
There have in the past been suggestions to strengthen this warning eg by injecting an audio warning while the mute is on, and some OS-specific tweaks (I think you can flash the application tab in the task bar on Windows maybe?). But it seems to be generally one of those "learned behaviour" things like removing the hand break when moving off in car - and doesn't bother enough people to warrant more signalling.
